packages/gensx/tests/execute.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { setTimeout } from "timers/promises";
+
+import { expect, suite, test } from "vitest";
+
+import { gsx, Streamable } from "../src";
+
+suite("execute", () => {
+  const WorkflowComponent = gsx.Component("test", () => "hello");
+
+  suite("workflow", () => {
```

Comment:
Should we have a test that runs two workflows in parallel and verifies that both get checkpoints?